src/lua/init.c

```diff
@@ -936,7 +936,8 @@ run_script_f(va_list ap)
 			goto luajit_error;
 		if (lua_main(L, argc, argv) != 0)
 			goto error;
-	} else if (!is_a_tty || (path && strcmp(path, "-") == 0)) {
+	} else if ((!interactive && !is_a_tty) ||
+			(path && strcmp(path, "-") == 0)) {
```

Comment:
How about removing `ERRINJ_STDIN_ISATTY` in a separate commit? It was added only because we had no option to enable the interactive mode unconditionally.

Reply:
(We can do it in a separate pull request.)

Reply:
PR #7555.